spec

```diff
@@ -545,7 +545,14 @@ beginseg
 #if OOT_DEBUG
     include "$(BUILD_DIR)/src/libultra/io/pfsfreeblocks.o"
 #endif
+#if PLATFORM_N64
+    include "$(BUILD_DIR)/src/libultra/os/getintmask.o"
+#endif
+#if PLATFORM_N64
+    include "$(BUILD_DIR)/src/libultra/gu/scale.o"
```

Comment:
merge these?

Reply:
I didn't because I think this way makes it clearer that gu/scale.o and mgu/scale.o are replacements for each other. Same with this bit:
```
#if PLATFORM_N64
    include "$(BUILD_DIR)/src/libultra/gu/mtxutil.o"
#endif
#if !PLATFORM_N64
    include "$(BUILD_DIR)/src/libultra/mgu/mtxidentf.o"
#endif
```
I didn't merge them because gu/mtxutil.o needs to be replaced with all of mgu/mtxf2l.o, mgu/mtxl2f.o, mgu/mtxidentf.o, and mgu/mtxindent.o. I'm not sure how much this matters though, the spec has become a bit of a mess anyway